cts/scheduler/xml/cancel-behind-moving-remote.xml

```diff
@@ -359,19 +359,19 @@
         <primitive class="ocf" id="ovndb_servers" provider="ovn" type="ovndb-servers">
           <instance_attributes id="ovndb_servers-instance_attributes">
             <nvpair id="ovndb_servers-instance_attributes-inactive_probe_interval" name="inactive_probe_interval" value="180000"/>
-            <nvpair id="ovndb_servers-instance_attributes-listen_on_master_ip_only" name="listen_on_master_ip_only" value="yes"/>
+            <nvpair id="ovndb_servers-instance_attributes-listen_on_promoted_ip_only" name="listen_on_master_ip_only" value="yes"/>
             <nvpair id="ovndb_servers-instance_attributes-manage_northd" name="manage_northd" value="yes"/>
-            <nvpair id="ovndb_servers-instance_attributes-master_ip" name="master_ip" value="172.17.1.87"/>
-            <nvpair id="ovndb_servers-instance_attributes-nb_master_port" name="nb_master_port" value="6641"/>
-            <nvpair id="ovndb_servers-instance_attributes-nb_master_protocol" name="nb_master_protocol" value="ssl"/>
+            <nvpair id="ovndb_servers-instance_attributes-promoted_ip" name="master_ip" value="172.17.1.87"/>
+            <nvpair id="ovndb_servers-instance_attributes-nb_promoted_port" name="nb_master_port" value="6641"/>
+            <nvpair id="ovndb_servers-instance_attributes-nb_promoted_protocol" name="nb_master_protocol" value="ssl"/>
```

Comment:
here too; these might change the outputs, not sure, but worth fixing in this PR?

Reply:
Probably would be an ordeal so never mind; I'm realizing there are a lot of names and such scattered around with it